plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Definition.hs

```diff
@@ -368,6 +371,30 @@ test_IdBuiltinList =
             typecheckEvaluateCekNoEmit def defaultBuiltinCostModelExt term @?=
                 Right (EvaluationSuccess xsTerm)
 
+test_BuiltinArray :: TestTree
+test_BuiltinArray =
+  testGroup "BuiltinArray" [
+    testCase "listToArray" do
+      let listOfInts = mkConstant @[Integer] @DefaultUni () [1..10]
+      let arrayOfInts = mkConstant @(Vector Integer) @DefaultUni () (Vector.fromList [1..10])
+      let term = apply () (tyInst () (builtin () ListToArray) integer) listOfInts
+      typecheckEvaluateCekNoEmit def defaultBuiltinCostModelForTesting term @?=
+          Right (EvaluationSuccess arrayOfInts)
```

Comment:
Some edge cases would be nice, like an empty array etc.

You should've probably used

```haskell
evals
    :: DefaultUni `HasTermLevel` a
    => a
    -> DefaultFun
    -> [Type TyName DefaultUni ()]
    -> [Term TyName Name DefaultUni DefaultFun ()]
    -> TestNested
```

from the below, we use it for unit tests. Not important though.

Reply:
> Some edge cases would be nice, like an empty array etc.

Those will definitely get tested in the conformance tests later.  Maybe it's  OK to have some overlap here though.